### PR TITLE
Add architecture image

### PR DESCRIPTION
**Pull request by junbeomlee**

### CODE REVIEW

**Comment by hihiboss** [comment]
How about insert “Execute” step in VM component?

**Comment by junbeomlee** [comment]
The corresponding figure shows the input and output of the component. It would be better to express it through another diagram, indicating what action`(Execute)` it is.

File: README.md [hunk 1]
```diff
@@ -3,3 +3,15 @@
 The project is inspired by [the simplicity](https://blockstream.com/simplicity.pdf) and the [ivy-bitcoin](https://github.com/ivy-lang/ivy-bitcoin).
 
 The koa project is to create a high-level language that has `more expressions` than the bitcoin script.
+
+
+
+### Architecture
```
Comment (zeroFruit) [inline README.md]: It may be trivial, but how about ## Architecture ?

**Comment by hihiboss** [comment]
@junbeomlee 
If another diagram expresses it, this wolud be nice!

**Pull request merged**